test/integration_test/applicationbackup_test.go

```diff
@@ -176,16 +181,106 @@ func createBackupLocation(
 			SecretConfig: newSecret.Name,
 		},
 	}
-	_, err = k8s.Instance().CreateBackupLocation(backupLocation)
-	return err
+	bkpLocation, err := k8s.Instance().CreateBackupLocation(backupLocation)
```

Comment:
`return k8s.Instance().CreateBackupLocation(backupLocation)`

Reply:
I don't see this fixed

Reply:
fixed.